### PR TITLE
feat: added cards component.

### PR DESCRIPTION
**Pull request by vaibhav-mahalle**
- Added Different type of card  components.

1. Text card 
2. Vertical card
3. Horizontal card
4. Card with Badge
5. Card with overlay

- Added the utility classes required for the card.
- Added the code snippets using Highlight js.

text card
Vertical card
badge card
overlay card

### CODE REVIEW

File: stylesheets/utilities.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    padding-bottom: 1rem;
+}
+
+.p-r-2{
+    padding-bottom: 1rem;
+}
+
+.p-2{
+    padding: 1rem;
+}
```
Comment (Sameony) [inline stylesheets/utilities.css]: fix the padding size, i think its supposed to be 2 rem

File: stylesheets/utilities.css [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+.p-l-3{
+    padding-bottom: 1rem;
+}
+
+.p-r-3{
+    padding-bottom: 1rem;
+}
+
+.p-3{
+    padding: 1rem;
```
Comment (Sameony) [inline stylesheets/utilities.css]: same issue

**Review by Sameony — APPROVED** [review]
Do follow up on the fixes before merging

File: stylesheets/utilities.css (hunk 1 above)
Comment (vaibhav-mahalle) [reply stylesheets/utilities.css]: fixed it.

File: stylesheets/utilities.css (hunk 2 above)
Comment (vaibhav-mahalle) [reply stylesheets/utilities.css]: fixed it.

File: stylesheets/utilities.css [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    padding-bottom: 1rem;
+}
+
+.p-l-1{
+    padding-bottom: 1rem;
+}
+
+.p-r-1{
+    padding-bottom: 1rem;
+}
```
Comment (sadathshariff) [inline stylesheets/utilities.css]: you have given the same style for different class

Comment (vaibhav-mahalle) [reply stylesheets/utilities.css]: fixed the mistake.

File: Documentation/cards/cards.html [hunk 4]
```diff
@@ -117,7 +117,360 @@
...
     <div class="parent-container">
-        
+        <h2 class="font-xl">Cards</h2>
+
+        <h3 class="font-md">Text Card</h3>
+        <div class="component-display-div">
+          <div class="flex-row flex-center">
+          <div class="card p-1">
+            <div class="card-heading-container">
+            <h3 class="font-md font-medium">Chitale Ice Cream</h3>
```
Comment (sadathshariff) [inline Documentation/cards/cards.html:127]: i think medium is for font-weight and md is for font-size, just a suggestion to have descriptive names

Comment (vaibhav-mahalle) [reply Documentation/cards/cards.html:127]: will keep that in mind

File: Documentation/cards/cards.html [hunk 5]
```diff
@@ -117,7 +117,360 @@
...
+            </div>
+          </code>
+        </pre>
+
+        <h3 class="font-md">Vertical Card</h3>
+        <div class="component-display-div">
+          <div class="flex-row flex-center">
+          <div class="card p-1">
+              <div class="image">
+                <img src="/assets/chitale ice cream.jpg" alt="">
```
Comment (sadathshariff) [inline Documentation/cards/cards.html]: try giving alt attribute some description

**Review by sadathshariff — COMMENTED** [review]
Suggested some  changes

**Review by sadathshariff — APPROVED** [review]

**Comment by vaibhav-mahalle** [comment]
- solved the issues pointed out by the reviewers
- approved by both

**Pull request merged**